src/docfx/Errors.cs

```diff
@@ -392,6 +392,9 @@ public static Error AtXrefNotFound(SourceInfo<string> source)
             public static Error XrefNotFound(SourceInfo<string> source)
                 => new Error(ErrorLevel.Warning, "xref-not-found", $"Cross reference not found: '{source}'.", source);
 
+            public static Error UidNotFound(string uid, IEnumerable<string?> repositories)
+                => new Error(ErrorLevel.Warning, "uid-not-found", $"Uid '{uid}' not found, which is referenced by repository {StringUtility.Join(repositories)}.");
```

Comment:
>Uid [](start = 68, length = 3)

change to "UID" to align to other errors

Reply:
OK